### DISCUSSION TITLE
Not working with ROG motherboard?

### DISCUSSION THREAD

**Discussion by mashirozx** [seerge/g-helper#1032] [Q&A]
I'm a desktop user, with ROG-STRIX-z790-a motherboard, ROG Power, Fan, GPU and other ROG fittings
![image](https://github.com/seerge/g-helper/assets/16148054/3f8569bd-943c-4198-b511-16e9f8c660ad)

When install the control interface, it said:
![image](https://github.com/seerge/g-helper/assets/16148054/6675a0db-4945-4c01-88a8-ee2c9f61265c)

Is this an expected behavior on a desktop machine?

**Comment by seerge** [comment]
@mashirozx hello, this is a laptop app as it states in a title :) 
Armoury for desktops is (de facto) absolutely different software (even tho under the same name)

P.S. If the only thing that you need is RGB controls, I would advice checking OpenRGB or SignalRGB projects
